source/_patterns/03-organisms/sections/timeline/timeline.yml

```diff
@@ -3,18 +3,24 @@ lead: Nullam at semper mauris, at rhoncus purus.
 timeline:
   - title: Create a settlement strategy
     excerpt: "<p>Before you enter into a negotiation, you should create a strategy as to how you will enter into negotiations.</p>"
-    read_more_link: "Strategy Details"
+    read_more:
+      label: "Strategy Details"
```

Comment:
You will need unique ids here, e.g. `timeline-full-modal-1`,  `timeline-full-modal-2`, etc.  That will fix [this](https://deploy-preview-89--confident-allen-d061ed.netlify.com/?p=organisms-timeline) and prevent accessibility errors when that is checked.